controller/test-files/event/list/ok-description.res.payload.golden.json

```diff
@@ -3,7 +3,10 @@
     {
       "attributes": {
         "name": "system.description",
-        "newValue": null,
+        "newValue": {
+          "content": "# Description is modified1",
+          "markup": "Markdown"
+        },
         "oldValue": null,
```

Comment:
@kwk Why does description have an oldValue while the rest of the fields don't?

Reply:
Oh, other non-relational fields DO have an old value.